index.html

```diff
@@ -2326,6 +2327,101 @@ <h2>Definition of Roles</h2>
 				</tbody>
 			</table>
 		</div>
+		<div class="role" id="comment">
+			<rdef>comment</rdef>
+			<div class="role-description">
+				<p>A comment contains a remark expressing a reaction to other content within the page.</p>
+				<p>The relationship of comments to other content SHOULD be exposed:</p>
```

Comment:
```suggestion
				<p>The relationship of comments to other content SHOULD be exposed as follows:</p>
```

(Just sort of feels like the "as follows" is missing).